lib/services/internalEngine/index.js

```diff
@@ -171,6 +171,7 @@ function InternalEngine (kuzzle) {
   this.create = function internalEngineCreate (type, id, content) {
     var
       request = {
+        refresh: true,
```

Comment:
I am not sure to agree on this.
If an application has some massive user writes, we do not want to issue a refresh each time.

I don't see any use case where within Kuzzle, we cannot control whether a manual refresh is needed or not.